src/Repository/CommitteeMembershipRepository.php

```diff
@@ -251,6 +251,8 @@ public function findMembers(Committee $committee): AdherentCollection
      * that does not already belongs to list of all members of $destinationCommittee
      *
      * NOTE: this returns poorly hydrated instances of Adherent, but is optimized for merging of committees
+     *
+     * @return Adherent[]|self
```

Comment:
Return doc seems not good: this method never returns `self`.

Reply:
fixed, thanks